Setup/Product.wxs

```diff
@@ -931,6 +941,14 @@
         <File Source="..\GitExtensions\Bin\$(var.Configuration)\ConEmu\GitShowBranch.cmd" />
       </Component>
     </DirectoryRef>
+    <DirectoryRef Id="DirThemes">
+      <Component Id="win10default.colors" Guid="*">
+        <File Source="..\GitExtensions\Bin\$(var.Configuration)\Themes\win10default.colors" />
+      </Component>
+      <Component Id="dark.colors" Guid="*">
+        <File Source="..\GitExtensions\Bin\$(var.Configuration)\Themes\dark.colors" />
```

Comment:
```suggestion
        <File Source="..\GitExtensions\bin\$(var.Configuration)\Themes\dark.colors" />
```

Reply:
Maybe we could create a variable containing "..\GitExtensions\bin\$(var.Configuration)" and use it everywhere...

It seems it could be done using:

    <?define BuildOutputPath= "..\GitExtensions\bin\$(var.Configuration)" ?>

Reply:
We are getting better at modifying the installer :)

Reply:
It's not straightforward, because translation-related files are added automatically